### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by immolsoft** [ntdevlabs/tiny11builder#285] [General]
Hello team, 
I have tried to create a tiny11 version of a Win Pro image (#7). All good except that the scripts strips IIS off too. AFAIK, IIS is one of the added features of Pro Vs Home. Would it be possible to have an option in the script to include it in the created image?

Thanks

**Comment by Elsewhereinthewoods** [comment]
Wonderful work, thanks! About the instructions on how to use, while it's specified to set the execution policy to unrestricted, I found out I still couldn't execute the script. Luckily I found out via a YouTube video I had to use the './' command in front of the 'tiny11maker.ps1'. 😉 Maybe include this info in the instructioins for others?

Alright, cheers, keep up the great work! 😊🙏🏻

**Comment by Miiraak** [comment]
Hi @immolsoft 

I don't think this will be added in the script.
But you can add IIS in `Turn Windows features on or off` control panel by checking the `Internet Information Services` checkbox.

or
 
you can use this command in cmd : 
```cmd
Start /w pkgmgr /iu:IIS-WebServerRole;IIS-WebServer;IIS-CommonHttpFeatures;IIS-StaticContent;IIS-DefaultDocument;IIS-DirectoryBrowsing;IIS-HttpErrors;IIS-ApplicationDevelopment;IIS-ASPNET;IIS-NetFxExtensibility;IIS-ISAPIExtensions;IIS-ISAPIFilter;IIS-HealthAndDiagnostics;IIS-HttpLogging;IIS-LoggingLibraries;IIS-RequestMonitor;IIS-Security;IIS-RequestFiltering;IIS-HttpCompressionStatic;IIS-WebServerManagementTools;IIS-ManagementConsole;WAS-WindowsActivationService;WAS-ProcessModel;WAS-NetFxEnvironment;WAS-ConfigurationAPI
```

**Comment by immolsoft** [comment]
Hi, thanks for your answer.
I have of course tried to enable IIS from turn features on/off. It does not
work and fails with a "catastrophic error".
I guess the error comes from that IIS has been stripped down from the
tiny11.iso. that's why I posted this feature request
Thanks

**Reply by Miiraak** [reply]
It's strange, i've test it before posting my answer and works fine for me... I was able to access the console
Did you have used coremaker ? 
Did the cmd command worked ?

**Reply by immolsoft** [reply]
Did you have used coremaker ? Yes I have
Did the cmd command worked ? Did not return any error but it didnt install anything

Couldn't it be because Windows Update is disabled in CoreMaker?

**Reply by Miiraak** [reply]
Yes is it, Windows update is desactivate due to supression of WinSxS and Windows update is used to add the Windows optional features.
You need to use `tiny11maker.ps1` or find another way to install it mabye [via the IIS package](https://www.microsoft.com/en-us/download/details.aspx?id=48264)
